app/uk.gov.hmrc.helptosavestridefrontend/controllers/StrideController.scala

```diff
@@ -110,7 +110,14 @@ class StrideController @Inject() (val authConnector:       AuthConnector,
   def customerNotEligible: Action[AnyContent] = authorisedFromStride { implicit request ⇒
     checkSession(
       SeeOther(routes.StrideController.getEligibilityPage().url),
-      whenIneligible = _ ⇒ Ok(views.html.you_are_not_eligible())
+      whenIneligible = { ineligible ⇒
+        IneligibilityReason.fromIneligible(ineligible).fold{
+          logger.warn(s"Could hot parse ieligiblity reason: $ineligible")
```

Comment:
Just these spelling mistakes then I'm happy to merge

Reply:
Updated!